app/assets/stylesheets/components/edit_profile.scss

```diff
@@ -581,79 +580,4 @@
       }
     }
   }
-}
-
-// LEGACY BELOW
```

Comment:
Should we open an issue to remove these, or are they unused?

Reply:
None of these classes are in use anymore. The bottom 4 were in the old edit profile page. The top #reset-password class was used in the reset your password page, but that style (dark background) is no longer needed as of this PR.

Reply:
Ah I think I got confused (thought they were commented out but still in the repo). Thanks for removing em!